packages/docs/pages/input.tsx

```diff
@@ -1,98 +1,147 @@
 import { Form, FormGroup, H1, Input, Panel, Text } from '@bigcommerce/big-design';
 import { CheckCircleIcon } from '@bigcommerce/big-design-icons';
-import React, { useState } from 'react';
+import React, { Fragment, useState } from 'react';
 
-import { Code, CodePreview, PageNavigation } from '../components';
+import { Code, CodePreview, ContentRoutingTabs, GuidelinesTable, List, NextLink } from '../components';
 import { InputPropTable } from '../PropTables';
 
 const InputPage = () => {
-  const items = [
-    {
-      id: 'examples',
-      title: 'Examples',
-      render: () => (
-        <>
-          <Panel>
-            <Text>Inputs are stylized form controls with the ability of controling validation.</Text>
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              {function Example() {
-                const [value, setValue] = useState('');
+  return (
+    <>
+      <H1>Input</H1>
 
-                const handleChange = (event) => setValue(event.target.value);
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Inputs</Code> are form controls that allow the entry of numeric or text data, often supported by
```

Comment:
Hmmm, trying to find a better phrasing here .. maybe "... that allow entering numeric or text data,... " ??

🤔 maybe others will have better suggestions, otherwise lgtm

Reply:
I'm good with either or 🤷‍♂️

Reply:
I'm also good with `allow the entry`, but I think I like better `that allow entering numeric or text data` @bigcommerce/dev-docs-team any suggestions? Thanks!

Reply:
Note: I updated `line 15` with the following suggestion: `that allow entering numeric or text data`